aztec_gddt/params.py

```diff
@@ -104,11 +225,16 @@
                                      fee_subsidy_fraction=1.0, # TODO
 
                                      # Phase Durations
-                                     phase_duration_proposal=10, # TODO
-                                     phase_duration_reveal=10, # TODO
-                                     phase_duration_commit_bond=10, # TODO
-                                     phase_duration_rollup=30, # TODO
-                                     phase_duration_race=30, # TODO
+                                     phase_duration_proposal_min_blocks=0, # TODO
```

Comment:
update numbers to reflect the workplan https://hackmd.io/@blockscience/B1ZQ_TX3T